datumaro/plugins/datumaro_format/converter.py

```diff
@@ -266,6 +271,38 @@ def _convert_cuboid_3d_object(self, obj):
         )
         return converted
 
+    def _convert_skeleton_object(self, obj: Skeleton) -> dict:
+        label_ordering = [
+            item["labels"]
+            for item in self.categories[AnnotationType.points.name]["items"]
+            if item["label_id"] == obj.label
+        ][0]
+
+        def get_label_order(label_id):
+            return label_ordering.index(
+                self.categories[AnnotationType.label.name]["labels"][label_id]["name"]
+            )
+
+        points = []
+        points_attributes = []
+        for element in sorted(obj.elements, key=lambda e: get_label_order(e.label)):
```

Comment:
It's not guaranteed that all the skeleton points are in the skeleton. The missing points should be considered having the `absent` visibility with default position 0, 0.

Reply:
Added processing for such a case, but now there is an assert that every element of Skeleton contains exactly one point